orbital-mechanics/Lecture7/Lecture7.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>May I also suggest that we get rid of "Lecture 7"; you will see that it doesn't read very nicely on the table of contents on the website:</p><p><a href="http://www.angadhn.com/SpacecraftDynamics/introduction.html" target="_blank">http://www.angadhn.com/SpacecraftDynamics/introduction.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>